db/db.go

```diff
@@ -423,14 +396,14 @@ func (ntc *nestedTableCache) LookupTable(nodeIP net.IP, t time.Time) (string, er
 	if err != nil {
 		return "", err
 	}
-	ntc.addTable(node.nodeName, tName)
+	ntc.addTable(node.name, tName)
 	return tName, nil
 }
 
 func (ntc *nestedTableCache) LookupNode(nodeIP net.IP) (*node, error) {
 	n, ok := ntc.nodes[nodeIP.String()]
 	if !ok {
-		return &node{}, fmt.Errorf("No such node")
+		return nil, fmt.Errorf("no such node")
```

Comment:
The dbCache's LookupNode methods return errNoNode instead of constructing a new err.